src/Table/Table.component.tsx

```diff
@@ -0,0 +1,266 @@
+// VENDOR
+import * as React from 'react';
+import classNames from 'classnames';
+import styled, { css } from '@xstyled/styled-components';
+import {
+    variant as createVariant,
+    margin as marginStyles,
+    MarginProps,
+    width as widthStyles,
+    WidthProps,
+    height as heightStyles,
+    HeightProps,
+    maxWidth as maxWidthStyles,
+    MaxWidthProps,
+    minWidth as minWidthStyles,
+    MinWidthProps,
+    padding as paddingStyles,
+    PaddingProps,
+    textAlign as textAlignStyles,
+    TextAlignProps,
+    fontSize as fontSizeStyles,
+    FontSizeProps,
+    fontWeight as fontWeightStyles,
+    FontWeightProps,
+    color as colorStyles,
+} from '@xstyled/system';
+
+interface SizeTheme {
+    [K: string]: {
+        cell?: any;
+        row?: any;
+        header?: any;
+
+        td?: any;
+        th?: any;
+    };
+}
+
+const TableSizes: SizeTheme = {
+    sm: {
+        th: { padding: '0.25rem 0.75rem' },
+        td: { padding: '0.25rem 0.75rem' },
+    },
+    md: {
+        th: { padding: '0.5rem 1.25rem 0.25rem' },
+        td: { padding: '0.5rem 1.25rem' },
+    },
+};
+
+export const sizeVariantStyles = createVariant({
+    key: 'table.sizes',
+    prop: 'size',
+    default: 'md',
+    variants: TableSizes,
+});
+
+// Table
+// ----------------------------------------------------------------------------------
+
+interface TableProps
+    extends React.HTMLAttributes<HTMLTableElement>,
+        MarginProps,
+        WidthProps,
+        MaxWidthProps,
+        MinWidthProps {
+    borderRadius?: string;
+    size?: string;
+}
+
+const StyledTable = styled('table')<TableProps>`
+    border: base;
```

Comment:
Shouldn't this be none? Or a prop?

Reply:
I would make `border` a prop, but keep `base` as the default.